index.js

```diff
@@ -612,6 +617,46 @@ class Args {
     // eslint-disable-next-line unicorn/no-process-exit
     process.exit();
   }
+
+  showAuthors() {
+    const parent = module.parent
+
+    // get author(s) from parent module
+    const {authors, author, collaborators} = parent.exports
+
+    const sublist = (title, list) =>
+      Array.isArray(list) ?
+        [
+          chalk.underline(title),
+          ...list.map(a =>
+            `${chalk.dim('-')} ${chalk.yellow(a)}`)
```

Comment:
Can you use the new print Methods `this.printMailColor()` and `this.printSubColor()`?
https://github.com/leo/args/blob/master/index.js#L33